projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/saved-cart/b2b-saved-cart.e2e-spec.ts

```diff
@@ -178,6 +205,50 @@ context('B2B - Saved Cart', () => {
           sampleData.savedActiveCartForm[0]
         );
       });
+
+      it('should make saved cart active from details page', () => {
+        savedCart.waitForSavedCartListingPageData([
+          sampleData.products[0],
+          sampleData.products[1],
+          sampleData.products[2],
+        ]);
+        savedCart.visitSavedCartListingPage();
+
+        cy.get('cx-saved-cart-list .cx-saved-cart-list-table tr')
+          .first()
+          .find('.cx-saved-cart-list-cart-id > .cx-saved-cart-list-value')
+          .then((cartId) => {
+            const cartCode = cartId.text();
+
+            savedCart.openFirstElementOnTheList(cartCode);
+            savedCart.manuallyRestoryCartFromDetailsPage(cartCode);
+            savedCart.verifyMiniCartQuantity(1);
+            savedCart.visitSavedCartListingPage();
```

Comment:
There's no point to have this as when you restore the saved cart from the details page, it automatically redirects you to the listing page